setup.py

```diff
@@ -75,7 +76,12 @@
         'Operating System :: Microsoft :: Windows',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```

Comment:
same as above, still have '3', does that means all 3.* is enabled?

Reply:
Nope, just same as docker-py, it would show in pip page:

![image](https://user-images.githubusercontent.com/5545529/67750935-1f97d680-fa6b-11e9-9d0b-dc8186c33dca.png)

Only this part would affect the installation, if user try to install it on Python 3.8, it would show 3.8 is not supported errors:
https://github.com/Azure/iotedgehubdev/pull/232/files#diff-2eeaed663bd0d25b7e608891384b7298R60

Reply:
Why we disable 3.1.*~3.4.*? Does we verified it not works, or it is because pywin32 limitation?https://pypi.org/project/pywin32/

Reply:
Because docker-py only support 3.5+